src/index.js

```diff
@@ -1 +1,16 @@
 import './style.css';
+import Scores from './modules/scores.js';
+
+const scores = new Scores();
+const refreshBtn = document.querySelector('.refresh-btn');
+const form = document.getElementById('form');
+
+scores.retrieveScores();
+
+refreshBtn.addEventListener('click', scores.retrieveScores);
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  scores.submitNewScore(form.name.value, form.score.value);
+  form.reset();
+});
```

Comment:
- [ ] It is required that when the `Refresh` button is clicked it hits the API endpoint and get all scores for a game created by you from the API (receiving data from the API and parsing the JSON). But the expected behaviour is not displayed at this moment. Kindly look into your logic that handles that and fix where the issue is.

### App behaviour at the moment.
[your-app-behaviour.webm](https://user-images.githubusercontent.com/26597432/219604606-4615ade1-9107-44dc-8f4f-5808fe81417b.webm)